.pipelines/singletenancy/dualstack-overlay/dualstackoverlay-e2e-step-template.yaml

```diff
@@ -65,6 +65,8 @@ steps:
       sudo go test -count=1 datapath_linux_test.go -timeout 3m -tags connection -run ^TestDatapathLinux$ -tags=connection,integration -isDualStack=true
       echo "Dualstack Overlay Linux datapath IPv4 test"
       sudo go test -count=1 datapath_linux_test.go -timeout 3m -tags connection -run ^TestDatapathLinux$ -tags=connection,integration
+      echo "cleaning up load-test namespace"
```

Comment:
Should this get changed to use make test-load now?

Reply:
This is a different test suite, this is under the test/integration/datapath directory, `make test-load` tests the files under `test/integration/load`, so I think we should leave it as is

Reply:
This is also in reference to the CNS `make test-integration` and restartCNS scenario that existed in the file before.